architecture/infrastructure_components/kubernetes_infrastructure.adoc

```diff
@@ -145,26 +189,26 @@ etcd.
 endif::[]
 
 [[node]]
-
 == Nodes
+
 A node provides the runtime environments for containers. Each node in a
-Kubernetes cluster has the required services to be managed by the
-xref:master[master]. Nodes also have the required services to run pods,
-including the Docker service, a xref:kubelet[kubelet], and a xref:service-proxy[service proxy].
+Kubernetes cluster has the required services to be managed by the master. Nodes
+also have the required services to run pods, including the *docker* service, a
```

Comment:
rather than say the docker service, can we just say including the container runtime?